src/index.js

```diff
@@ -46,18 +58,21 @@ module.exports = (
 
   const getFilesList = directory => {
     let packageJsonFound = false;
-    const folderFiles = fs.readdirSync(directory);
-    const sandboxFiles = folderFiles
+    const folderFiles = [];
+    const sandboxFiles = [];
+    getFiles(directory, directory, folderFiles);
+
+    folderFiles
       // we ignore the package.json file as it will
       // be handled separately
       .filter(file => file !== 'package.json')
       .map(file => {
         const fullFilePath = path.resolve(directory, file);
         const content = fs.readFileSync(fullFilePath, 'utf-8');
-        return {
+        sandboxFiles.push({
           name: file,
           content,
-        };
+        });
       });
```

Comment:
And then you can just do

```js
    const sandboxFiles = getFiles(directory)
      // we ignore the package.json file as it will
      // be handled separately
      .filter(file => file !== 'package.json')
      .map(file => {
        const fullFilePath = path.resolve(directory, file);
        const content = fs.readFileSync(fullFilePath, 'utf-8');
        return {
          name: file,
          content,
        };
      });
```